plugins/woocommerce/client/blocks/assets/js/blocks/product-filters/inner-blocks/attribute-filter/block.json

```
@@ -92,5 +92,6 @@
 		"attributes": {
 			"isPreview": true
 		}
-	}
+	},
+	"viewStyle": "woocommerce/product-filter-attribute-view-style"
```

Was is supposed to be `viewStyle` or `style`? `viewStyle` are explicitly loaded on the frontend only and `style` in frontend AND editor.

```suggestion
	"style": "woocommerce/product-filter-attribute-view-style"
```